contentScript.js

```diff
@@ -315,18 +319,16 @@ const createHrSeparator = () => {
 const createForm = () => {
 
   // basic styles : reused
-  let textLabelStyles = "color: black; padding-left: 1%;";
-  let inputStyles = "color: black; background: white;";
-  let warnLabelStyles = "color: red; padding-left: 1%;";
+  let textLabelStyles = "color: black";
+  let warnLabelStyles = "color: red";
 
   // parent div for form
   let wrapperDiv = document.createElement("div");
   wrapperDiv.id = "formWrapper";
-  wrapperDiv.style = "position: fixed; background: white; top: 12.5%; width: 75%; left: 12.5%; border: 3px solid #73AD21; overflow:scroll; height: 80%"
 
   // mobile number input field
   let mobileinputid = "data-mob";
-  let mobileInput = createInput(mobileinputid, inputStyles, "number", mobilenumber);
+  let mobileInput = createInput(mobileinputid, "", "number", mobilenumber, 'form-control');
```

Comment:
I love this. Love how you didn't deform everything, thanks.

Reply:
Thanks!